pymatgen/analysis/diffraction/xrd.py

```diff
@@ -130,9 +130,13 @@ def __init__(self, wavelength="CuKa", symprec=0, debye_waller_factors=None):
         """
         if isinstance(wavelength, float):
             self.wavelength = wavelength
-        else:
+        elif isinstance(wavelength, int):
+            self.wavelength = float(wavelength)
```

Comment:
I think int and float can be handled in one if statement. The subsequent calculations don't really care whether it si a float or an int.

Reply:
Thanks, that sounds good.  I pushed another commit which brings the two together.